modules/services/dropbox.nix

```
@@ -6,11 +6,11 @@ let
 
   cfg = config.services.dropbox;
   baseDir = ".dropbox-hm";
-  dropboxCmd = "${pkgs.dropbox-cli}/bin/dropbox";
+  dropboxCmd = "${pkgs.dropbox}/bin/dropbox";
   homeBaseDir = "${config.home.homeDirectory}/${baseDir}";
 
 in {
-  meta.maintainers = [ maintainers.eyjhb ];
+  meta.maintainers = [ maintainers.tph5595 ];
```

Since you are not in the Nixpkgs repo maintainer list, you will have to add yourself to the `modules/lib/maintainers.nix` file and change this line to

```suggestion
  meta.maintainers = [ hm.maintainers.tph5595 ];
```

No need to add real name if you feel uncomfortable with that, your GitHub user ID is 6002071:

```console
$ curl -s https://api.github.com/users/tph5595 | jq .id
6002071
```